### DISCUSSION TITLE
How to use Hot Design when a ContentDialog is open?

### DISCUSSION THREAD

**Discussion by sarosigy** [unoplatform/studio#7] [General]
The title is pretty much the question. How to access Hot Design features when a ContentDialog is open?

First observation: The HotDesign toolbar is not show on the surface of a ContentDialog, and the one on the Main Window is inaccessible, obviously.
Second observation: While Hot Design open, pressing the play and cause to open a ContentDialog in the app, the dialog's overlay and the dialog itself will be over the whole Hot Design surface, and only the ContentDialog can be clicked on, nothing else.

**Comment by nickrandolph** [comment]
@sarosigy thanks for providing this feedback. In the first iteration of HotDesign there isn't ContentDialog support but we are tracking this in our backlog. 
There's no easy workaround but one option would be to encapsulate the contents of the dialog in a usercontrol and then include the usercontrol in page of the application (just for the purposes of developing the UI using HotDesign).

**Reply by DevTKSS** [reply]
just for completness: @sarosigy this is listed below #66 meanwhile as known issue
